docs/Design/Classes/Class Declarations.md

```diff
@@ -6,18 +6,20 @@ This improvement adds support for class declarations to GLS, including:
 
 * Naming a class
 * Declaring any generics *(optional)*
-* Declaring a class to extend *(optional)*
+* Declaring a parent class to extend *(optional)*
+* Declaring parent interfaces to implement *(optional)*
 
 
 ## Commands
 
 ### `class start`
 
-`class start` `:` `classDescriptor`*`[, parentClassDescriptor]`*
+`class start : classDescriptor [ extends parentClassDescriptor] [ implements parentInterfaceName1 parentInterfaceName2 parentInterfaceName3 ...] `
```

Comment:
The spacing around `[`s here is actually important. As currently set, the text implies that including the `parentClassDescriptor` requires two spaces.

```
class start : classDescriptor  extends parentClassDescriptor
                             ^^
```